app/views/candidate_interface/unsubmitted_application_form/show.html.erb

```diff
@@ -18,31 +18,24 @@
         (<%= EndOfCycleTimetable.find_reopens.to_s(:govuk_date) %>). You can keep making changes to the rest of your application until then.
       </p>
     </section>
-    <% elsif @application_form.candidate_has_previously_applied? %>
-      <% if @application_form.previous_application_form.application_choices.rejected.any?  %>
+    <% else %>
+      <% if @application_form.candidate_has_previously_applied? && @application_form.previous_application_form.application_choices.rejected.any? %>
         <%= render(CandidateInterface::RejectionReasonsComponent.new(application_form: @application_form.previous_application_form)) %>
       <% end %>
-      <% if @application_form.apply_1? %>
+      <% if  @application_form.candidate_has_previously_applied? && !@application_form.apply_1? %>
```

Comment:
Do we need the `@application_form.candidate_has_previously_applied?` here?

Reply:
Yup! Done